### ISSUE TITLE
Auto Theatre Mode not working after latest update

### ISSUE THREAD

**Issue by Kyrazhi** [night/betterttv#5382]
I had been experiencing the auto-theatre mode option not working for the past while. I'm on Firefox, Windows 10 if that helps.

I saw that there was a fix put out within the past couple of days so I made sure to update the add-in, and I even restarted both my browser and computer just in case. But I'm still getting the auto-theatre mode option not working.

When I open up a stream, it'll open with a black screen either in normal mode or in theatre mode, but then it'll quickly open into theatre mode and then back into normal mode before the stream actually starts.

I've tried toggling the option in settings a few times, closing the tab while it's in theatre mode or not. None of it has helped it stick.

I saw #5361 was the issue that had the fix, and that #5379 was closed yesterday for being "already fixed", but it hasn't been fixed for me and I definitely have the update.

**Comment by Kotaless** [comment]
It works when I refresh the page. I belive it wasn't fixed properly.

**Comment by Cybernick5** [comment]
Auto-Theatremode dont works for me either. deleted bttv many times, cleared chache and refreshed the browser many times. No chance xD

**Comment by grenijs** [comment]
For me it works for offline channels (which pretty much breaks the page) and doesnt work for live channels.
PLS FIX VOLVO! :@

**Comment by night** [comment]
This is fixed as of the latest update.

**Comment by Kyrazhi** [comment]
@night I'm sorry, but this was not fixed as of the latest update, that's what I said in my post. The automatic theatre mode setting is not working at all, I've tried several different configurations of refreshing the page, toggling the setting on and off, and made sure several times that I actually have the latest update.

This needs to be re-opened or, if it has been fixed, then someone needs to tell me how to actually apply the fix because it's clearly not working.
